src/amo/components/SearchForm/index.js

```diff
@@ -119,10 +123,22 @@ export class SearchFormBase extends React.Component {
       filters.addonType = addonType;
     }
 
-    dispatch(autocompleteStart({
-      errorHandlerId: errorHandler.id,
-      filters,
-    }));
+    if (
+      // See: https://github.com/moroshko/react-autosuggest#onsuggestionsfetchrequested-required.
+      reason === 'input-focused' &&
+      deepEqual(filters, this.state.lastAutocompleteFilters)
+    ) {
+      log.debug(oneLine`Ignoring suggestions fetch requested because it has been
```

Comment:
I think this could be a bit clearer/concise, maybe just:

"No autocomplete start dispatched: search input focused but filters have not changed."